src/App/ProjectDetail/ProjectDetailController.java

```diff
@@ -184,6 +184,18 @@ private void onreleaseID(KeyEvent keyEvent) {
     private void showDays(ActionEvent event) throws IOException {
         getEsti_time();
 
+        if (start_date.getValue() != null) {
+            final Date startDate = Date.valueOf(start_date.getValue());
+            Date endDate = Date.valueOf(end_date.getValue());
+            if (endDate.compareTo(startDate) < 0) {
+                Label label = new Label("End date must be equal or greater than start date");
+                JFXPopup popup = new JFXPopup(label);
```

Comment:
The popup message is blocking the End Date JFXDatePicker which is not good to look at. I would rather suggest you to to place a label field below the end date field in [Projectdetail FXML file](https://github.com/atiqueahmedziad/Easy-Project-Management-Tool/blob/master/src/App/ProjectDetail/projectdetail.fxml)
And once you get the difference between start date and end date less than 0, you can set the label text to ```End date must be equal or greater than start date```

<img width="741" alt="screenshot 2018-10-30 at 7 39 40 pm" src="https://user-images.githubusercontent.com/8364578/47721992-94c7a480-dc7b-11e8-8da9-24e429ff34c5.png">

Reply:
Do you think it should have any color to call the **users attention**?